content/200-concepts/100-components/250-preview-features/080-cli-preview-features.mdx

```diff
@@ -5,18 +5,12 @@ metaDescription: Prisma CLI features that are currently in preview.
 
 <TopBlock>
 
-The following Preview features are available for the Prisma CLI:
-
-- [`db seed`](../../../reference/api-reference/command-reference#db-seed-preview)
-
 To use a Preview CLI feature, [use the `--preview-feature` flag](./) when you run the command. [Share your feedback on all Preview features on GitHub](https://github.com/prisma/prisma/issues/3108).
 
-</TopBlock>
+<Admonition type="info">
 
-## Enabling a Prisma CLI preview feature
+There are currently no Preview CLI features.
 
-To use a Prisma CLI Preview feature, append the `--preview-feature` flag each time you run the command:
+</Admonition>
 
-```terminal
-npx prisma db seed --preview-feature
-```
```

Comment:
Maybe keep this in, at least commented out, for when we add a CLI preview feature flag again.